person.rb

```diff
@@ -32,6 +33,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
```

Comment:
- [ ] Kindly create `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental` 👍 They should be implemented similar to the following ⬇️

**Step 1: Kindly add the following method in your book.rb file 👍**
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

**Step 2: Kindly make your constructor of rental class something like below ⬇️**

```ruby
def initialize(date, book, person)
  @book = book
  book.rentals << self
  @person = person
  person.rentals << self
end
```

**Step 3: Kindly add the following method in your person.rb file 👍**

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```